### ISSUE TITLE
SELinux Policy not working on EL8.5

### ISSUE THREAD

**Issue by Oats87** [rancher/rke2-selinux#24]
```
time="2021-11-15T19:43:11.343394657+01:00" level=error msg="copy shim log" error="read /proc/self/fd/20: file already closed"
time="2021-11-15T19:43:11.350153887+01:00" level=error msg="RunPodSandbox for &PodSandboxMetadata{Name:etcd-redacted,Uid:5035b6cfd6d86b57b6d465fc038c1d82,Namespace:kube-system,Attempt:0,} failed, error" error="failed to create containerd task: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: failed to set /proc/self/attr/keycreate on procfs: write /proc/self/attr/keycreate: invalid argument: unknown"
```

Versions:

```
Installed Packages
rke2-common.x86_64                     1.21.6~rke2r1-0.el8                    @rke2
rke2-selinux.noarch                    0.8-2.el8                              @rke2-common
rke2-server.x86_64                     1.21.6~rke2r1-0.el8                    @rke2
```

For reference: https://rancher-users.slack.com/archives/C01PHNP149L/p1637002191151000

**Comment by dweomer** [comment]
Should be fixed by https://github.com/rancher/rke2-selinux/pull/21 which is currently available via the `testing` channel.

**Comment by jonaz** [comment]
@dweomer can we have this published? Our clusters just broke with 8.5 :(

**Comment by galal-hussein** [comment]
closing the issue, since this issue has been resolved
